apps/barista-design-system/src/pages/overview-page/shared/check-badge-type.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export function checkBadgeType(
+  badge: string[],
+): { icon: string; style: string } | undefined {
+  let _badge: { icon: string; style: string } | undefined = undefined;
```

Comment:
don't use `_` for normal variables – why do you not create a base class to extend from? (setter, getter etc.)
```suggestion
  let badge: { icon: string; style: string } | undefined = undefined;
```

Reply:
I can't see the benefit of that right now but maybe I am missing something. I'm trying that out right now